lib/entry/LiveEntry.js

```diff
@@ -130,6 +131,30 @@ function saveFlavorDownloadersConfiguration() {
     });
 }
 
+function getEntryServerNodeId() {
```

Comment:
the method getLiveEntryServerNodes can also accept a filter of type KalturaEntryServerNodeFilter which you can put serverTypeEqual = this.entryServerType, then you know the entryServerNode you receive is the correct one and it will simplify this method.